lumigator/backend/backend/api/routes/jobs.py

```diff
@@ -52,7 +52,17 @@ def create_inference_job(
     response: Response,
     background_tasks: BackgroundTasks,
 ) -> JobResponse:
-    job_response = service.create_job(job_create_request, background_tasks)
+    job_response = service.create_job(job_create_request)
+
+    async def background_task(job_response: JobResponse, job_create_request: JobInferenceCreate):
```

Comment:
I'd say not to use local-scope functions unless the enclosing scope is really needed, and maybe use lambdas in that case. Passing params to module-scoped funs should be enough in most cases. I'm not sure what's the lifecycle of local-scope functions; we might be stuck with lots of copies of the same fun over and over that don't get GC'ed???